db/migrate/20150724143942_set_completed_at_field.rb

```diff
@@ -0,0 +1,9 @@
+class SetCompletedAtField < ActiveRecord::Migration
+  def up
+    Item.completed.update_all 'completed_at = updated_at'
+  end
+
+  def down
+    Item.completed.update_all 'completed_at = null'
```

Comment:
Btw, the previous one was perfect: `completed_at: null`. I suggested sql text in order to refer to appropriate column of the row and not just pass static value for `up` method of the migration.